File: C#/04-Operadores Relacionais/Relacionais.cs
using System;

class Relacional
{
    static void Main(string[] args)
    {
    //Os operadores relacionais em C# são:
    // > maior < menor >= maior ou igual =< menor ou igual 
    // == igual != diferente    

        int a = 1;
        int b = 3;
        int c = 1;
        bool teste;//Vai mostrar relação testada

    //Vamos testar a relação entre estas variaveis
        Console.WriteLine("Dadas variaveis " +a +" " +b +" " +c); 
        Console.WriteLine("------------------------------------------------------");
        teste = a > b;
        Console.WriteLine(a +" maior que > " +b +" = " +teste);//Falso
        teste = a < b;
        Console.WriteLine(a +" menor que < " +b +" = " +teste);//Verdadeiro
        teste = a <= c;
        Console.WriteLine(a +" menor ou igual <= " +c +" = " +teste);//Verdadeiro
        teste = a >= b;
        Console.WriteLine(a +" maior ou igual >= " +b +" = " +teste);//Falso
        teste = a == b;
        Console.WriteLine(a +" igual == " +b +" = " +teste);//Falso
        teste = a != b;
        Console.WriteLine(a +" diferente != " +b +" = " +teste);//Verdadeiro
        Console.ReadLine();
       

    }
}

File: Java/04-Operadores Relacionais/Relacionais.java
class Relacionais {
    public static void main(String[] args) {
        /*
         * Os operadores relacionais em java são: <(menor) >(maior) <=(menou ou igual)
         * >=(maior ou igual) ==(igual) !=(diferente) o teste retorna true(verdadeiro)
         * ou false(falso)
         */

        int a = 1;
        int b = 4;
        int c = 1;
        boolean teste;

        // vamos testar a relação entre as variavies
        teste = a > b;
        System.out.println(a + " > maior que " + b + " ? " + teste);// false

        teste = a < b;
        System.out.println(a + " < menor que " + b + " ? " + teste);// true

        teste = a >= b;
        System.out.println(a + " >= maior ou igual " + b + " ? " + teste);// false

        teste = a <= c;
        System.out.println(a + " <= menor ou igual " + c + " ? " + teste);// true

        teste = a == b;
        System.out.println(a + " == igual a " + b + " ? " + teste);// false

        teste = a != b;
        System.out.println(a + " != diferente de " + b + " ? " + teste);// true
    }
}